backend/review/management/commands/precompute_pcr_views.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from django.core.cache import cache
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.http import Http404
+from tqdm import tqdm
+
+from courses.models import Topic
+from review.models import CachedReviewResponse
+from review.views import manual_course_reviews
+
+
+def precompute_pcr_views(verbose=False, is_new_data=False):
+    if verbose:
+        print("Now precomputing PCR reviews.")
+
+    CachedReviewResponse.objects.all().update(expired=True)
```

Comment:
This is not in an atomic block, so what's the potential time in which the cached responses are down totally. Is that acceptable?

Reply:
NVM i see what you're doing

Reply:
`expired` CachedReviewResponses can still be returned from the view

Reply:
Yeah, but I think there's no reason to not have this in an atomic block either – will change.